ios/Sources/HomeFeature/QuestionnaireView.swift

```diff
@@ -0,0 +1,43 @@
+import Styleguide
+import SwiftUI
+
+public struct QuestionnaireView: View {
+    public let onTapAnswer: () -> Void
+
+    public init(onTapAnswer: @escaping () -> Void) {
+        self.onTapAnswer = onTapAnswer
+    }
+
+    public var body: some View {
+        VStack(alignment: .trailing, spacing: 12) {
+            HStack {
+                Image(uiImage: AssetImage.logo.image)
+                Text(L10n.HomeScreen.Questionnaire.title)
+                    .foregroundColor(Color(AssetColor.Base.primary.color))
```

Comment:
Maybe it's better to create extension to directly access SwiftUI.Color by editing `swif5.stencil`, I recommend make issue for this 😄